### ISSUE TITLE
Accessibility Feedback - Design Intent- Public Websites Team - Outreach and Events

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#32301]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- Thanks for the thorough briefing! I understand that the date filter is the first in a number of iterations. This feedback contains suggestions for things to look at going forward.

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
List and Pagination UI must be consistent with other list and pagination UI. Like number of results and page count, etc.

Users must be aware of other available times below the register button. Either they need to be moved above the register button or have some other way to access them. After accessing them, navigating back to the register button should be simple

**Should:**

**Consider:**

**Comment by FranECross** [comment]
@laflannery This ticket is back isn't in Icebox anymore, and I'm wondering if it should go back to Icebox, or if it's obsolete and I can close? Thanks in advance!

**Comment by laflannery** [comment]
@FranECross I would recommend closing this. This was from quite a long time ago and the 2 must items I would imagine are covered:
1. Use standard paging UI: We are using the v3 pagination component
2. Other available times should be easily accessible: I can't see how this was proposed vs what we have now but we have these in an accordion component on the detail page.

**Comment by FranECross** [comment]
Closing per comment above. (Thanks Laura!)
